lib/VeresOne.js

```diff
@@ -274,11 +277,23 @@ class VeresOne {
     this.logger.log('Sending to ledger, operation type:', operation.type);
     const {didDocument} = options;
 
+    let targetNode;
+    try {
+      targetNode = await targetNodeClient.getTargetNode(
+        {hostname: this.hostname});
+    } catch(e) {
+      console.log('Error in getTargetNode.', e.message);
```

Comment:
Does this need to be logged?  If so, better to use the `this.logger`?  (Does it have an `error()` method?)

Reply:
fixed.